```
Run bin/mailmap_check.py
This author is not included in the .mailmap file:
Sai Udayagiri <saibabu.udayagiri@email.com>
This author is not included in the .mailmap file:
saiudayagiri <144413200+saiudayagiri@users.noreply.github.com>

The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.

        For instructions on updating the .mailmap file see:
https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#mailmap-instructions

The following authors will be added to the AUTHORS file at the time of
the next SymPy release.

	Pasquale Cocchini <pasquale.cocchini@gmail.com>
	Sai Udayagiri <saibabu.udayagiri@email.com>
	saiudayagiri <144413200+saiudayagiri@users.noreply.github.com>
Error: Process completed with exit code 1.
```

--- a/.mailmap
+++ b/.mailmap
@@ -185,6 +185,7 @@
 #
 # See also: 'git shortlog --help' and 'git check-mailmap --help'.
 #
+
 *Chu-Ching Huang <cchuang@mail.cgu.edu.tw>
 *Dan <coolg49964@gmail.com>
 *Marc-Etienne M.Leveille <protonyc@gmail.com>
@@ -1336,6 +1337,7 @@ Sagar Bharadwaj <sagarbharadwaj50@gmail.com>
 Sagar231 <sagarfeb298@gmail.com>
 Sahil Shekhawat <sahilshekhawat01@gmail.com>
 Sai Nikhil <tsnlegend@gmail.com>
+Sai Udayagiri <saibabu.udayagiri@gmail.com> Sai Udayagiri <saibabu.udayagiri@email.com>
 Saicharan <62512681+saicharan2804@users.noreply.github.com>
 Saicharan <saicharanhahaha@gmail.com>
 Saikat Das <saikatdchhe@gmail.com>

--- a/.mailmap
+++ b/.mailmap
@@ -186,6 +186,7 @@
 # See also: 'git shortlog --help' and 'git check-mailmap --help'.
 #
 
+
 *Chu-Ching Huang <cchuang@mail.cgu.edu.tw>
 *Dan <coolg49964@gmail.com>
 *Marc-Etienne M.Leveille <protonyc@gmail.com>
@@ -1337,6 +1338,7 @@ Sagar Bharadwaj <sagarbharadwaj50@gmail.com>
 Sagar231 <sagarfeb298@gmail.com>
 Sahil Shekhawat <sahilshekhawat01@gmail.com>
 Sai Nikhil <tsnlegend@gmail.com>
+Sai Udayagiri <saibabu.udayagiri@gmail.com> <144413200+saiudayagiri@users.noreply.github.com>
 Sai Udayagiri <saibabu.udayagiri@gmail.com> Sai Udayagiri <saibabu.udayagiri@email.com>
 Saicharan <62512681+saicharan2804@users.noreply.github.com>
 Saicharan <saicharanhahaha@gmail.com>

--- a/.mailmap
+++ b/.mailmap
@@ -1550,7 +1550,6 @@ Tschijnmo TSCHAU <tschijnmotschau@gmail.com>
 Tuan Manh Lai <laituan245@gmail.com>
 Tuomas Airaksinen <tuomas.airaksinen@gmail.com>
 Tyler Pirtle <teeler@gmail.com>
-Udayagiri Saibabu <saibabu.udayagiri@gmail.com> Sai Udayagiri <saibabu.udayagiri@gmail.com>
 Unknown <kunda@scribus.net>
 Uwe L. Korn <uwelk@xhochy.com>
 V1krant <46847915+V1krant@users.noreply.github.com>

